environments/kubernetes/manifests/thanos-store-statefulSet.yaml

```diff
@@ -95,7 +95,6 @@ spec:
           requests:
             cpu: 32m
             memory: 64Mi
-      volumes: null
```

Comment:
@metalmatze Do you think this change would create a problem?

Reply:
I don't think so, as this has been null before. But I'm not 100% sure.

Reply:
I think it should be equivalent.